src/modules/new-request-form/translations/en-us.yml

```diff
@@ -0,0 +1,134 @@
+title: "Copenhagen Theme New Request Form"
+packages:
+  - "new-request-form"
+parts:
+  - translation:
+      key: "new-request-form.answer-bot-modal.title"
+      title: "Title for the Answer Bot modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "While you wait, do any of these articles answer your question?"
+  - translation:
+      key: "new-request-form.answer-bot-modal.view-article"
+      title: "Label for the link to view a suggested article in the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "View article"
+  - translation:
+      key: "new-request-form.answer-bot-modal.footer-title"
+      title: "Title of the footer of the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "Does this article answer your question?"
+  - translation:
+      key: "new-request-form.answer-bot-modal.footer-content"
+      title: "Title of the footer of the Answer Bot Modal"
+      screenshot: "https://drive.google.com/file/d/14qsEyUiVTy9MRYBGkgVWLmlduhoscYBw/view?usp=drive_link"
+      value: "If it does, we can close your recent request #{{requestId}}"
```

Comment:
Is the hash sign part of the placeholder?
If not, could it be part of the placeholder? This will ensure the hash sign will be stay on the left even in RTL languages (see: https://app.screencast.com/WI0uMzGftFPvA) :) 
Doc: https://zendesk.atlassian.net/wiki/spaces/globalization/pages/618696752/RTL+and+Signs

Reply:
> For RTL languages the translator can just put the # sign on the right.

@Fredx87 The thing is that we have a validation in Rosetta preventing us from changing the # sign's position, and we can't bypass it or remove it completely :/
![image](https://github.com/zendesk/copenhagen_theme/assets/100944954/c9acc83f-9068-4ae6-b9c0-e6012e07e937)

> If you instead prefer to move it to the placeholder for coherence we can do that as well. In this case, we need to be sure that the placeholder is rendered in the code using the special Unicode characters mentioned in the docs, right?

Yes could we please move it to the placeholder? That's what we have been asking other teams to do as well :) And yes we would need to use the special Unicode character
If that helps here's an example PR: https://github.com/zendesk/zendesk_console/pull/34346/files

Reply:
Make sense. I removed the `#` sign from the string and moved it to the placeholder.

Reply:
Thank you very much! :)